recipes/imagl/all/conanfile.py

```diff
@@ -58,13 +58,17 @@ def config_options(self):
     def requirements(self):
         if self.options.with_png:
             self.requires("libpng/1.6.37")
+        if self.options.with_jpeg and tools.Version(self.version) >= "0.2.0":
+            self.requires("libjpeg/9d")
 
     def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self)
         self._cmake.definitions["STATIC_LIB"] = not self.options.shared
         self._cmake.definitions["SUPPORT_PNG"] = self.options.with_png
+        if tools.Version(self.version) >= "0.2.0":
```

Comment:
Might be worth add a property since this will be required 3 times

```py
@property
def _supports_jpeg:
    return tools.Version(self.version) >= "0.2.0"
```

Reply:
Done in my last commit